bool isDriver
int16 driverID
